coursefinder/templates/coursefinder/course_finder_base.html

```diff
@@ -1,45 +1,48 @@
 {% extends "base.html" %}
-{% load static %}
-{% load wagtailcore_tags %}
-{% load discover_uni_tags %}
+{% load static wagtailcore_tags discover_uni_tags sass_tags %}
 
 {% block extra_css %}
-<link rel="stylesheet" href="{% static 'css/coursefinder.css' %}">
+    <link href="{% sass_src 'scss/course_finder.scss' %}" rel="stylesheet" type="text/css" >
 {% endblock extra_css %}
 
 {% block content %}
     <div class="course-finder-base">
-        <div class="header">
-            <h2>
-                {% get_translation key='course_finder' language=page.get_language %}
-            </h2>
-        </div>
+        <div class="discover-uni-container">
+            <ul class="course-finder-base__nav">
+                {% if page.back_page %}
+                <a href={% pageurl page.back_page %} class="course-finder-base__nav-control">
+                    <li class="course-finder-base__nav-control-back">
+                        <img class="course-finder-base__nav-control-icon" src="{% static 'images/chevrons/green_left_chevron.svg' %}" alt="">
+                        {% get_translation key='back' language=page.get_language %}
+                    </li>
+                </a>
+                {% endif %}
 
-        <ul class="navigation">
-            {% if page.back_page %}
-                <a href={% pageurl page.back_page %}><li class="helper-text back">{% get_translation key='back' language=page.get_language %}</li></a>
-            {% endif %}
+                <!--<a href={% pageurl page.get_parent %} onclick="handleStartAgain()">-->
```

Comment:
I left this in until I did the results page, as I didn't know if it would be needed there.
The results page was done in the next PR, this wasn't needed and  was removed then.